zerver/actions/invites.py

```diff
@@ -72,7 +72,7 @@ def too_many_recent_realm_invites(realm: Realm, num_invitees: int) -> bool:
         # Very high entropy realm names are suspicious
         warning_flags.append("random-realm-name")
 
-    if not realm.description:
+    if realm.description == Realm.DEFAULT_REALM_DESCRIPTION:
         warning_flags.append("no-realm-description")
 
```

Comment:
I don't think we need to add a `no-realm-description` if we provide a default description, but not sure.

Reply:
I changed it to `default-realm-description`. Is that OK?

Reply:
Hmm, ya I guess better to wait for maintainer to comment